go.mod

```diff
@@ -6,6 +6,7 @@ go 1.16
 require (
 	github.com/google/certificate-transparency-go v1.1.1
 	github.com/google/go-attestation v0.3.2
+	github.com/google/go-cmp v0.5.5
```

Comment:
any reason we add this dependency over just using `reflect.DeepEqual`? All the fields are exported in CosTlv and we don't override equality checks.

Reply:
I was following the [style guide](https://g3doc.corp.google.com/go/g3doc/style/decisions.md?cl=head#types-of-equality), but seems like using DeepEqual is fine here, and can remove a dependency?

Reply:
Ah if it's explicitly called out, then let's migrate everything over, ideally in another PR.

Reply:
Okay, so I will keep using cmp here. And change all reflect.DeepEqual to cm.Equal in next PR.